src/pages/journal/components/empty-slot/__mocks__/screen-orientation.mock.ts

```diff
@@ -0,0 +1,34 @@
+import { Observable } from 'rxjs/Observable';
```

Comment:
We already have a mock at - src/pages/journal/components/test-slot/__mocks__/screen-orientation.mock.ts

Could we not reuse/improve that one

Reply:
or do what ammar said ;)

Reply:
I accidentally duplicated the screen orientation mock - have removed and using the original.